CromIAM/src/test/scala/cromiam/webservice/QuerySupportSpec.scala

```diff
@@ -70,24 +70,28 @@ class QuerySupportSpec extends FlatSpec with Matchers with ScalatestRouteTest wi
 
   "GET query" should "peacefully forward to Cromwell if nothing is untoward" in {
     Get(getQuery).withHeaders(authHeaders) ~> queryGetRoute ~> check {
+      contentType should be(ContentTypes.`text/plain(UTF-8)`)
```

Comment:
not sure why these are `text/plain`?

Reply:
Talked IRL. Has to do with the mock transitively using [`HttpEntity.apply(string: String)`](https://github.com/akka/akka-http/blob/v10.1.7/akka-http-core/src/main/scala/akka/http/scaladsl/model/HttpEntity.scala#L276). These entities end up with a mime-type of `text/plain(UTF-8)`.